strr-web/components/bcros/form-section/contact-information/Form.vue

```diff
@@ -203,13 +203,13 @@ onMounted(() => {
 const form = ref()
 
 watch(form, () => {
-  if (form.value && isComplete) { form.value.validate() }
+  if (form.value && isComplete) { form.value.validate({ silent: true }) }
```

Comment:
@shaangill025 this removed the validation error in console and fixed the issue with tests failures.